### DISCUSSION TITLE
Lego Tonie Box

### DISCUSSION THREAD

**Discussion by nifix777** [Wendelpunkt/ToypadChallenge#7] [Ideas]
Als alternative zur Toniebox mit z.B. einem Raspi dem das Pad angeschlossen wird.
Wird eine Figur auf das Pad (die zentrale Position) gestellt wird eine Musikdatei auf dem Raspi abgespielt.
Dazu kann man das Pad in Farben leuchten lassen.

**Comment by tomwendel** [comment]
Love the idea!

**Comment by pherting** [comment]
Fühl ich auch!
